godtools/ViewControllers/MenuViewController.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  MenuViewController.swift
+//  godtools
+//
+//  Created by Devserker on 4/24/17.
+//  Copyright © 2017 Cru. All rights reserved.
+//
+
+import UIKit
+
+class MenuViewController: BaseViewController, UITableViewDataSource, UITableViewDelegate {
+    
+    @IBOutlet weak var tableView: UITableView!
+    let menuCellIdentifier = "cellIdentifier"
+    let general = ["language_settings", "about", "help", "contact_us", "notifications", "preview_mode_translators_only"]
+    let share = ["share_god_tools", "share_a_story_with_us"]
+    let legal = ["terms_of_use", "privacy_policy"]
+    let header = ["menu_general", "menu_share", "menu_legal"]
+    
+    override var screenTitle: String {
+        get {
+            return "settings".localized
+        }
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.setupStyle()
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
+    
+    // MARK: UI
+    
+    fileprivate func setupStyle() {
+        self.view.backgroundColor = .clear
+        self.tableView.backgroundColor = .gtGreyLight
+        self.registerCells()
+    }
+    
+    // MARK: - Navigation Buttons
+    
+    override func configureNavigationButtons() {
+        self.addEmptyLeftButton()
```

Comment:
This isn't a big deal, so we don't need to spend a bunch of time on it. I was mainly curious.
One last thing, did you try `self.navigationItem.hidesBackButton = true` ? If you did and that didn't work, let's keep this since it works and move on :)